config/rubocop.yml

```diff
@@ -19,15 +19,14 @@ Style/StringLiterals:
 Metrics/LineLength:
   Max: 100
```

Comment:
I like the strategy, but if we lower it below 100 we will have approx 20 more excludes, so I will leave this for a separate decision by @backus.